routes/merged/github.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch("github") do |r|
+    r.on web? do
+      r.get "callback" do
```

Comment:
Roda also supports to combine matchers, right?

```suggestion
      r.get web?, "callback" do
```

Reply:
Yep.  I didn't check that this was the only route in the file.  I'll make this change.